recipes-support/simple-http-server/files/simple-http-server.service

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=Simple built-in http server in python
+Wants=network.target
+
+[Service]
+Type=oneshot
+ExecStart=/usr/sbin/simple-http-server.sh
```

Comment:
You shouldn't `pushd` in the script in the first place, just a `cd ...`.  `popd` here makes no difference since you are telling systemd to change directory.